### DISCUSSION TITLE
Converting multiple Anime file to show in Paraview

### DISCUSSION THREAD

**Discussion by Serince** [OpenRadioss/OpenRadioss#496] [Show and tell]
I have written a small script to convert multiple anime files and csv file to vtk files to open in paraview. Simply, extract the file and **put the analysis folder** and run there. 
[paraview.tar.gz](https://github.com/OpenRadioss/OpenRadioss/files/9607143/paraview.tar.gz)

**Comment by MarianBulla** [comment]
This is great !
Thank you very much for this. 
Works very well.

**Reply by ZdendaH** [reply]
Thank you. I need it.

**Comment by PaulAltair** [comment]
Thanks! I have used the guts of your script to create some 'doitall' runscripts (I removed the section where the converters are compiled)

3 scripts (runradfile.sh, runkfile.sh and runkeyfile.sh) to run _0000.rad, .k and .key input respectively

These take your filename and a number 'n' as input to run a job on n processors mpi, autoconvert the Annn files to vtk and T01 to csv, then load those files into paraview

e.g. `./runradfile.sh tensile_LAW36_BIQUAD_0000.rad 2` would run the job on 2 processors, convert results and open them in paraview

read the readme, it contains more detailed notes (paths to update etc)

[runxxxfile.tar.gz](https://github.com/OpenRadioss/OpenRadioss/files/9890440/runxxxfile.tar.gz)

[runxxxfile_README.md](https://github.com/OpenRadioss/OpenRadioss/files/10079729/runxxxfile_README.md)

**Comment by FGarelli** [comment]
Does anyone have a windows version of the script?

**Reply by PaulAltair** [reply]
These scripts would be relatively easy to modify to bat files, but there is already a Windows version of the GUI submission tools I created, including vtk and csv conversion, download and instructions here: [GUI Job Submission Guide in Confluence](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/45252609/python+tk+guis+for+job+submission+anim-vtk+conversion+and+T01-csv+conversion) that might be easier still

See Also: https://github.com/orgs/OpenRadioss/discussions/1231

**Reply by FGarelli** [reply]
Hi Paul, the script looks amazing but unfortunaly I cannot use them.
For the running one I obtain the following error:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/b8ceba91-1a7a-4cb7-8830-3186a31b0656)
For the vtk conversion i receive the following:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/c62cf89c-d956-4b92-a480-c56bef8858f0)
All the directories doesn't have any spaces inside the name. Any ideas?

**Reply by PaulAltair** [reply]
Did you put the scripts folder inside your OpenRadioss installation and launch it from there? (folder should go at base level, next to exec, hmcfg folders) for the MPI script, you need to point to your mpivars file, it might be easier to try the 'non mpi' version first?

**Reply by FGarelli** [reply]
Hi Paul,
yes, the script's folder is at the same level of the other folders that you mentioned.
<img width="626" alt="image" src="https://github.com/OpenRadioss/OpenRadioss/assets/132582318/c96bfbb7-2756-4925-a7e0-3067c72f3566">
I tried both the version and the vtk conversion (that is actually the main time-saving script that i would like to implement) but I find the same problems.

**Reply by PaulAltair** [reply]
hmm, you are building the win version from source? 
I haven't tried with a locally built version, but I'm not sure why it would make a difference

it looks like the exe files aren't finding the bat files at all, i.e. the message isn't even there saying 'Anim File Being Converted is' but I'm not sure how that is possible as the bat file should be in same path

I have tried breaking my install by e.g. putting files in wrong place etc, and I can't replicate how it is failing in your case.

If you look in the bat file, you can see the code that is being run is very simple, all the script is doing is allowing you to pick multiple files and submit them all sequentially, other than that, it is just running the anim-vtk script, so you should be able to debug line by line where the issue might be
![image](https://github.com/OpenRadioss/OpenRadioss/assets/98418895/66297281-ea55-4a74-b9cc-9c3e7fbe94d7)

**Reply by FGarelli** [reply]
The problem was with the directory name of the file to be converted. The script worked until the level with parenthesis inside the name of the folder (it may be that the problem).
I moved the folder on another directory and it works!

**Reply by PaulAltair** [reply]
ah great, thanks for reporting back, I tested, and can confirm it is not working if there are parentheses in the path
